.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,29 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
+{
+	"name": "Python 3",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": ".."
+	},
+	"features": {
+		"ghcr.io/devcontainers/features/aws-cli:1": {},
+		"ghcr.io/jungaretti/features/make:1": {}
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"ms-azuretools.vscode-docker",
+				"ms-vscode.makefile-tools",
+				"ms-python.black-formatter",
+				"tamasfe.even-better-toml",
+				"lfm.vscode-makefile-term"			]
+		}
+	},
+
+	// Mounts the host .aws folder into the container to allow profile access
+	"mounts": [
+		"source=${env:HOME}/.aws,target=/home/vscode/.aws,type=bind" // macOS, Linux, WSL
```

Comment:
If I make changes to my host `.aws` files how do they get propagated to the container? Do I need to rebuild it?

Reply:
It's a bind mount, so there is a two way sync

Reply:
It's also mounted outside the git repo scope, so won't be in source control